core/src/main/java/io/substrait/relation/ProtoRelConverter.java

```diff
@@ -2,10 +2,7 @@
 
 import static io.substrait.expression.proto.ProtoExpressionConverter.EMPTY_TYPE;
 
-import io.substrait.expression.AggregateFunctionInvocation;
-import io.substrait.expression.Expression;
-import io.substrait.expression.FunctionLookup;
-import io.substrait.expression.ImmutableExpression;
+import io.substrait.expression.*;
```

Comment:
Revert the wildcard for imports

Reply:
Sorry `intelliJ` default setting of conversion to import.* after 5 imports.